docs/reference/api/resources/command.rst

```diff
@@ -68,11 +68,6 @@ a similar pattern.
 It doesn't matter what order you add commands to the app - the group, section and order
 will be used to display the commands in the right order.
 
-If a command is added to a toolbar, it will automatically be added to the app
-as well. It isn't possible to have functionality exposed on a toolbar that
-isn't also exposed by the app. So, ``cmd2`` will be added to the app, even though
-it wasn't explicitly added to the app commands.
-
```

Comment:
Unsure why this has been deleted. It's a fairly important property of commands.

The key reason for this is discoverability - the menu always contains all the commands that the app can run. They may be disabled if they're not applicable, but they're always *present* (and thus discoverable). The toolbar is just a way for the the UI designer to make a menu option easier to access.

Reply:
I removed this paragraph because it wasn't actually behaving this way on any platform, but on closer inspection I see you must have broken it by accident. I've fixed it, added a test, and restored the documentation.